ux.symfony.com/README.md

```diff
@@ -2,48 +2,63 @@
 
 Source code for [ux.symfony.com](https://ux.symfony.com).
 
-## Contributing
+## Installation
 
-### Local Development
+### Source code
 
-1. Install the project:
-    ```bash
-    git clone git@github.com:symfony/ux
-    cd ux/ux.symfony.com/
-    ```
+Install the project:
+```bash
+git clone git@github.com:symfony/ux
+cd ux/ux.symfony.com/
+```
 
-2. Install the dependencies:
-    ```bash
-    symfony composer install
-    ```
+Install the dependencies:
+```bash
+symfony composer install
+```
 
-3. (optional) Configure Docker:
-    ```bash
-    docker compose up -d
-    ```
+### Services
 
-5. Run database migrations:
-    ```bash
-    symfony console doctrine:migration:migrate
-    ```
+(optional) Configure Docker to launch Mercure
+```bash
+docker compose up -d
+```
 
-6. Populate the database:
-    ```bash
-    symfony console app:load-data
-    ```
+### Database
 
-7. Compile the Sass files:
-    ```bash
-    symfony console sass:build --watch
-    ```
+Run database migrations:
+```bash
+symfony console doctrine:migration:migrate
+```
+
+Populate the database:
+```bash
+symfony console app:load-data
+```
 
-8. Start the web server:
-    ```bash
-    symfony server:start -d
-    ```
+### Assets
 
+Download the importmap packages locally:
+```bash
+symfony console importmap:install
+```
```

Comment:
We can also omit this step if we update the recipes so that it takes place during the composer installation process: https://github.com/symfony/recipes/blob/main/symfony/asset-mapper/6.4/manifest.json#L13C31-L13C31

Reply:
Yes great idea, i’ll update it then

Reply:
Done, thank you!